packages/desktop-client/src/components/budget/IncomeHeader.tsx

```diff
@@ -1,11 +1,16 @@
-import React from 'react';
+import React, { type ComponentProps, type ReactNode } from 'react';
 
 import Button from '../common/Button';
 import View from '../common/View';
 
 import RenderMonths from './RenderMonths';
 
-function IncomeHeader({ MonthComponent, onShowNewGroup }) {
+type IncomeHeaderProps = ComponentProps<typeof RenderMonths> & {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we don't need `RenderMonths` props here.

```suggestion
type IncomeHeaderProps = {
```